topics/metagenomics/tutorials/plasmid-metagenomics-nanopore/tutorial.md

```diff
@@ -0,0 +1,514 @@
+a	utg000001l	0	channel_364_204a2254-2b6f-4f10-9ec5-6d40f0b870e4_template:101-4457	+	4357
+```
+
+## Remapping using Minimap2
+
+The Assembly graph created can be used for mapping again with minimap2, but first the graph should be transformed to FASTA format. Remapping is done on the miniasm assembly to map the original reads on the assembly to provide the nucleotides per position. This is used by **Racon** {% icon tool %} for consensus construction.
+
+> ### {% icon hands_on %} Hands-on: Pairwise sequence alignment
+>
+> 1. **GFA to Fasta** {% icon tool %} with the following parameters
+>   - *"Input GFA file"*: the `Assembly Graph` (collection) created by the Miniasm tool
+>
+> 2. **Map with minimap2** {% icon tool %} with the following parameters
+>    - *"Will you select a reference genome from your history or use a built-in index?"*: `Use a genome from history and build index`
+>    - *"Use the following dataset as the reference sequence"*: `FASTA file` collection created by **GFA to Fasta** {% icon tool %}
+>    - *"Single or Paired-end reads"*: `single`
+>    - *"Select fastq dataset"*: The `Plasmids` collection
+>    - *"Select analysis mode (sets default)"*: `PacBio/Oxford Nanopore read to reference mapping (-Hk19)`
+>    - In the section **Set advanced output options**:
+>       - *"Select an output format"*: `paf`
+>
+>
+>     {% include snippets/select_collection.md %}
+>
+>     > ### {% icon question %} Question
+>     >
+>     > How many contigs do we have for the RB05 sample after the use of **Minimap2** {% icon tool %} and **Miniasm** {% icon tool %}?
+>     > <br><br>
+>     > Hint: run **Nanoplot** {% icon tool %} on the output of **GFA to Fasta** {% icon tool %}
+>     >
+>     > > ### {% icon solution %} Solution
+>     > > 22
+>     > >
+>     > > This can be determined by looking at the NanoStats output of NanoPlot.
+>     > {: .solution }
+>     {: .question}
+{: .hands_on}
+
+## Ultrafast consensus module using Racon
+
+The mapped reads can be improved even more using **Racon** {% icon tool%} ({% cite Vaser2017 %}) to find a consensus sequence.
+Racon is a standalone consensus module to correct raw contigs generated by rapid assembly methods which do not include a consensus step.
+Racon generates genomic consensus which is of similar or better quality compared to the output generated by assembly methods which employ both error correction and consensus steps.
```

Comment:
That is a rather harsh claim. Is there a reference. If not can you make it sound less strong.

Reply:
These evaluations all mainly depend on the complexity of the organism and their genomes, so I would say better to just refer to one of the benchmarking papers for further studies: e.g. these two https://doi.org/10.1101/530824 and https://doi.org/10.1093/bib/bbx147

Reply:
thanks for the references @mmiladi will update